### PR TITLE
LPS-195920 Migrate Display Template Selector to clay Picker

### PR DESCRIPTION
**Pull request by DiegoHu97**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by DiegoHu97** [comment]
ci:test:sf

File: modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/configuration/display_settings.jsp [hunk 1]
```diff
@@ -7,19 +7,22 @@
 
 <%@ include file="/init.jsp" %>
 
-<%
-PortletURL configurationRenderURL = (PortletURL)request.getAttribute("configuration.jsp-configurationRenderURL");
-%>
-
 <div class="display-template">
-	<liferay-template:template-selector
```
Comment (ealonso) [inline modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/configuration/display_settings.jsp:15]: @DiegoHu97 you should do it in liferay-template:template-selector not only for asset publisher, you should modify the taglib

Comment (ealonso) [reply modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/configuration/display_settings.jsp:15]: In that way your changes are going to be available in all portal

**Pull request closed without merging**